src/style/themes/themes.spec.js

```diff
@@ -0,0 +1,158 @@
+import mergeWithBase from './base';
+import classicTheme from './classic';
+import genericTheme from './generic';
+import smallBusinessTheme from './small_business';
+import mediumBusinessTheme from './medium_business';
+import largeBusinessTheme from './large_business';
+import colorConfig from '../color-config';
+import generatePalette from '../palette';
+import atOpacity from '../utils/at_opacity';
+
+const isObject = (obj) => {
+  return typeof obj === 'object' && obj !== null;
+};
+
+const assertIsSubset = (obj, comparison) => {
+  if (!isObject(obj)) {
+    // no further nesting, assert that values are equal
+    expect(obj).toEqual(comparison);
+    return;
+  }
+
+  const objKeys = Object.keys(obj);
+  const comparisonKeys = Object.keys(comparison);
+
+  objKeys.forEach((key) => {
+    // assert that keys are present
+    expect(comparisonKeys.includes(key)).toBeTruthy();
+
+    // repeat for nested objects
+    assertIsSubset(obj[key], comparison[key]);
+  });
+};
```

Comment:
These tests don't test any scenario where keys are overridden - only that new keys are added.

Reply:
There is a spec on the merge-deep util that does this. Should I repeat this here?

Reply:
I didn't see that, thats fine - except the merge-deep spec doesn't test values are overridden either

Reply:
I've since made changes so that it does.